firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -37,3 +37,37 @@ void centerPiece()
     // Turn electromagnet off
     digitalWrite(ELECTROMAGNET, LOW);
 }
+
+// The handler function that acts on different error codes output by movement primitives
```

Comment:
A better way to phrase comments before a function definition would be to start with a verb, and explain what it does, also with any constraints it has. That might look like this: 
```
// Handles behavior based on different error codes passed in
// Must take in an error code to work properly, returns true by default
```